### PR TITLE
LRQA-77336 SF

### PR DESCRIPTION
**Pull request by KevinQueiroz**

### CODE REVIEW

**Comment by KevinQueiroz** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LRQA-77336 Test 3 - CanDeleteBlog</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LRQA-77336">Test 3 - CanDeleteBlog</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Subtarefa</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Média</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>Aberto</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Support QA</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Alex Candido</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

1 file changed, 76 insertions(+)

**Comment by KevinQueiroz** [comment]
ci:test:sf

File: teste-t5/teste.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+teste commit + pull request
```
Comment (ViniLopes20) [inline teste-t5/teste.txt:1]: Oi @KevinQueiroz, você poderia remover esse commit?

**Review by ViniLopes20 — APPROVED** [review]
Muito bom @KevinQueiroz, obrigado por isso 😄 

PS: Somente remova aquele outro commit para seu SF passar

**Comment by ViniLopes20** [comment]
Lembre de atualizar seu ticket quando pegar algo para trabalhar e depois mover ele para review quando finalizar:

https://issues.liferay.com/browse/LRQA-77336

**Pull request closed without merging**